File: src/com/leetcode/Easy/MakeTwoArraysEqualByReversingSubArrays.java
package com.leetcode.Easy;

import java.util.*;

// 1460. Make Two Arrays Equal by Reversing Sub-arrays
public class MakeTwoArraysEqualByReversingSubArrays {
    public boolean canBeEqual(int[] target, int[] arr) {
        if (target.length != arr.length)
            return false;

        Arrays.sort(target);
        Arrays.sort(arr);

        for (int i = 0; i < arr.length; i++) {
            if (target[i] != arr[i])
                return false;
        }
        return true;
    }
}

File: JavaScript/Easy/MakeTwoArraysEqualByReversingSubarrays.js
/**
 * 1460. Make Two Arrays Equal by Reversing Subarrays
 * @param {number[]} target
 * @param {number[]} arr
 * @return {boolean}
 */
const canBeEqual = function (target, arr) {
  arr.sort((a, b) => a - b);
  target.sort((a, b) => a - b);
  for (let i = 0; i < target.length; i++) {
    if (target[i] !== arr[i]) {
      return false;
    }
  }

  return true;
};